app/client/src/features/events/Moderation/ManageQuestions/CurrentQuestionCard.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @typescript-eslint/indent */
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Chip, Card } from '@material-ui/core';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+
+import type { useQuestionQueueFragment$key } from '@local/__generated__/useQuestionQueueFragment.graphql';
+import { QuestionAuthor, QuestionContent } from '../../Questions';
+import { NextQuestionButton } from './NextQuestionButton';
+import { PreviousQuestionButton } from './PreviousQuestionButton';
+import { useQuestionQueue } from './useQuestionQueue';
+import { useRecordPush } from './useRecordPush';
+import { useRecordRemove } from './useRecordRemove';
+import { useRecordUnshift } from './useRecordUnshift';
+import { useEnqueuedPush } from './useEnqueuedPush';
+import { useEnqueuedRemove } from './useEnqueuedRemove';
+import { useEnqueuedUnshift } from './useEnqueuedUnshift';
+
+const useStyles = makeStyles((theme) => ({
+    item: {
+        width: '100%',
+        paddingTop: theme.spacing(0.5),
+        borderRadius: '10px',
+    },
+    relative: {
+        position: 'relative',
+        overflow: 'visible',
+        marginTop: theme.spacing(3)
+    },
+    answeringNow: {
+        padding: theme.spacing(0.5),
+        position: 'absolute',
+        top: theme.spacing(-2),
+        left: '50%',
+        transform: 'translateX(-50%)',
+        background: '#F5C64F',
+        color: 'white',
```

Comment:
theme.palette.background.default (so we can support dark mode in the future)

https://mui.com/customization/default-theme/